lib/active_model/serializer/adapter/json.rb

```diff
@@ -4,44 +4,122 @@ class ActiveModel::Serializer::Adapter::Json < ActiveModel::Serializer::Adapter
 
         def serializable_hash(options = nil)
           options ||= {}
+          serialized_hash = {}
```

Comment:
I'd call this one `hash` instead of `serialized_hash` since the method is supposed to return a serializ*able* hash, not a serializ*ed* hash. What do you think?